### PR TITLE
Align tick elements to baseline grid

### LINKED ISSUE #2186: Radio buttons and checkboxes look misaligned
**Describe the bug**
Radio buttons have different spacing then checkboxes, so placing them close to each other make them look misaligned.

screenshot 2019-03-05 at 15 52 56

**To Reproduce**
1. Create a simple form with both checkboxes and radio buttons (one under another)
2. Radios and checkboxes have different spacing from left and different spacing between button and label, so they look misaligned

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Checkboxes, radio buttons and their labels are not aligned to the baseline grid. This PR corrects that.

It also adds a class called ```.is-inline-label```, which removes the padding top (nudge that aligns text to the baseline grid). This is useful when you want to align the label to text that is not wrapped in a tag, e.g. ```<td>text here</td>```

In addition, it fixes #2186 

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/templates/tick-element-comparison/
- Turn baseline grid on
- Verify all text sticks to the baseline grid

The example above also shows that the vertical space between checkboxes / radios matches the space between paragraph and button.

## Details

[List of links to issues/bugs and cards if needed]

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@kwm14 @deadlight please hold off with the review on this. I need to convert rem values to pixels, because border-radius doesn't work predictably with fractional values and distorts the circles.

**Comment by deadlight** [comment]
@lyubomir-popov I've taken off the labels, put them back on when you're ready for the review

**Comment by lyubomir-popov** [comment]
@deadlight ready for review

File: examples/templates/tick-element-comparison.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          <th class="u-align--right">Disks</th>
+          <th class="u-align--right">Storage(GB)</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th aria-label="FQDN | MAC">
+            <form>
+              <input type="checkbox" id="checkExample1" checked="">
+              <label for="checkExample1" class="is-inline-label">label</label>
```
Comment (deadlight) [inline examples/templates/tick-element-comparison.html]: There's a name conflict here - change the name of the checkbox

Comment (lyubomir-popov) [reply examples/templates/tick-element-comparison.html]: @deadlight nice catch, thanks. fixed.

**Comment by lyubomir-popov** [comment]
image

**Comment by lyubomir-popov** [comment]
@percy shows a discrepancy between baselines of checkboxes and radios on one hand, and selects etc on the other: 
image

chrome looks ok:
image

**Comment by kwm14** [comment]
Design LGTM 👍

**Pull request merged**